src/dtos/monitor-default.dto.ts

```diff
@@ -168,33 +162,39 @@ export class MonitorDefaultBaseDTO {
     },
   )
   @IsString()
-  @ValidateIf(o => o.defaultPurposeCode || o.endDate === null || o.endHour === null)
+  @IfIsAtive(isNotEmpty, {}, {
+    message: (args: ValidationArguments) => {
+      return CheckCatalogService.formatResultMessage('DEFAULT-51-A', {
+        fieldname: args.property,
+        key: KEY,
+      });
+    },
+  })
   defaultPurposeCode: string;
```

Comment:
When attempting to import an inactive _monitoring default_ record, this still fails if the `defaultPurposeCode` is set to an empty string: this seems to be due to the `@IsInDbValues` decorator, which marks empty values as invalid. We may want to discuss this, but I think for the checks where empty values are allowed for inactive records, we should update `IsInDbValues` to pass through empty values (like `IsValidCode` does), or we should use the initially proposed `ValidateIf` logic instead.